packages/protocol/contracts/governance/Governance.sol

```diff
@@ -4,8 +4,8 @@ import "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/math/Math.sol";
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-import "solidity-bytes-utils/contracts/BytesLib.sol";
 
+import "./Proposals.sol";
```

Comment:
How are we doing on `Governance` bytecode size now?

Reply:
<22 kb